Source/WebCore/platform/graphics/controls/ControlStyle.h

```diff
@@ -52,11 +52,13 @@ struct ControlStyle {
         ReadOnly            = 1 << 14,
         ListButton          = 1 << 15,
         ListButtonPressed   = 1 << 16,
+        VerticalWritingMode = 1 << 17,
     };
     OptionSet<State> states;
     unsigned fontSize { 12 };
     float zoomFactor { 1 };
     Color accentColor;
+    Color visitedDependentColor;
```

Comment:
I would call this `textColor`, since it comes from `CSSPropertyColor`.

`visitedDependentColor` doesn't make sense, because there are a number of colors that can be dependent on the visited state.

Reply:
The name was changed.